package.json

```diff
@@ -9,61 +9,61 @@
   "bugs": "https://github.com/merkle-open/env-linter/issues/new",
   "commitlint": {
     "extends": [
-      "@namics/commitlint-conventional-changelog"
+      "@merkle-open/commitlint-conventional-changelog"
     ]
   },
   "config": {
     "commitizen": {
-      "path": "@namics/cz-conventional-changelog"
+      "path": "@merkle-open/cz-conventional-changelog"
     }
   },
   "devDependencies": {
-    "@commitlint/cli": "16.3.0",
-    "@namics/commitlint-conventional-changelog": "1.2.2",
-    "@namics/cz-conventional-changelog": "2.0.3",
-    "@namics/eslint-config": "9.1.1",
-    "@namics/prettier-config": "2.0.1",
-    "@namics/ts-config": "1.1.1",
-    "@types/fs-extra": "9.0.13",
-    "@types/jest": "27.5.1",
-    "@types/node": "17.0.33",
-    "@types/node-fetch": "2.6.1",
-    "@types/semver": "7.3.9",
-    "@types/zen-observable": "0.8.3",
+    "@commitlint/cli": "18.4.2",
+    "@merkle-open/commitlint-conventional-changelog": "1.0.6",
+    "@merkle-open/cz-conventional-changelog": "1.1.0",
+    "@merkle-open/eslint-config": "3.0.0",
+    "@merkle-open/prettier-config": "1.2.0",
+    "@merkle-open/ts-config": "1.1.0",
+    "@types/fs-extra": "11.0.4",
+    "@types/jest": "29.5.8",
+    "@types/node": "20.9.1",
+    "@types/node-fetch": "2.6.9",
+    "@types/semver": "7.5.5",
+    "@types/zen-observable": "0.8.6",
     "chalk": "4.1.2",
     "ci-info": "3.3.1",
     "clean-webpack-plugin": "4.0.0",
     "commander": "9.2.0",
-    "commitizen": "4.2.4",
-    "eslint": "7.32.0",
-    "eslint-plugin-import": "2.26.0",
+    "commitizen": "4.3.0",
+    "eslint": "8.54.0",
+    "eslint-plugin-import": "2.29.0",
     "execa": "5.1.1",
     "find-packages": "8.0.13",
     "fs-extra": "10.1.0",
     "globby": "11.1.0",
-    "husky": "7.0.4",
-    "jest": "27.5.1",
+    "husky": "8.0.3",
+    "jest": "29.7.0",
     "license-checker": "25.0.1",
-    "lint-staged": "12.4.1",
+    "lint-staged": "15.1.0",
     "log-symbols": "4.1.0",
     "node-fetch": "2.6.7",
-    "npm-check-updates": "12.2.0",
+    "npm-check-updates": "16.14.11",
     "npm-run-all": "4.1.5",
-    "prettier": "2.6.2",
-    "rimraf": "3.0.2",
-    "semver": "7.3.7",
+    "prettier": "3.1.0",
+    "rimraf": "5.0.5",
+    "semver": "7.5.4",
     "standard-version": "9.5.0",
     "terminal-link": "2.1.1",
     "ts-config-webpack-plugin": "2.0.3",
-    "ts-jest": "27.1.4",
-    "tslib": "2.4.0",
-    "typescript": "4.6.4",
-    "webpack": "4.46.0",
+    "ts-jest": "29.1.1",
+    "tslib": "2.6.2",
+    "typescript": "5.3.2",
+    "webpack": "4.47.0",
     "webpack-cli": "4.9.2",
-    "zen-observable": "0.8.15"
+    "zen-observable": "0.10.0"
   },
   "engines": {
-    "node": ">=14.15.0"
+    "node": ">=16.20.0"
```

Comment:
This is a breaking change and must not be hidden in a dev dependencies update.
What are we doing?